nextflow_schema.json

```
@@ -1162,6 +1229,95 @@
                 }
             }
         },
+        "souporcell_options": {
+            "title": "Souporcell options",
+            "type": "object",
+            "fa_icon": "fas fa-users",
+            "description": "Options specific to the Souporcell tool for clustering mixed-genotype scRNAseq experiments by individual.",
+            "properties": {
+                "souporcell_ploidy": {
+                    "type": "integer",
+                    "description": "Ploidy, must be 1 or 2.",
+                    "default": 2,
+                    "enum": [1, 2],
+                    "fa_icon": "fas fa-dna"
+                },
+                "souporcell_min_alt": {
+                    "type": "integer",
+                    "description": "Min alt to use locus.",
+                    "default": 10,
+                    "minimum": 0,
+                    "fa_icon": "fas fa-arrow-down"
+                },
+                "souporcell_min_ref": {
+                    "type": "integer",
+                    "description": "Min ref to use locus.",
+                    "default": 10,
+                    "minimum": 0,
+                    "fa_icon": "fas fa-arrow-down"
+                },
+                "souporcell_max_loci": {
+                    "type": "integer",
+                    "description": "Max loci per cell, affects speed.",
+                    "default": 2048,
+                    "minimum": 0,
+                    "fa_icon": "fas fa-arrow-up"
+                },
+                "souporcell_restarts": {
+                    "type": "integer",
+                    "description": "Number of restarts in clustering, when there are > 12 clusters we recommend increasing this to avoid local minima.",
+                    "default": 100,
+                    "minimum": 0,
+                    "fa_icon": "fas fa-redo"
+                },
+                "souporcell_common_variants": {
+                    "type": "string",
+                    "format": "file-path",
+                    "exists": true,
+                    "pattern": "^\\S+\\.vcf(\\.gz)?$",
+                    "description": "Common variant loci or known variant loci vcf, must be vs same reference fasta.",
+                    "default": null,
+                    "fa_icon": "fas fa-file-code"
+                },
+                "souporcell_fasta": {
```

```suggestion
                "fasta": {
```

Also move this field to 'Reference genome options'

---

Adopted!